tests/test_wrapper_writer.py

```diff
@@ -0,0 +1,189 @@
+from unittest import TestCase
+
+import os
+
+import shutil
+
+from wrapper_writer.container import Container
+from wrapper_writer.method import Method
+from wrapper_writer.structure import Structure
+from wrapper_writer.wrapper_writer import WrapperWriter
+
+
+class TestWrapperWriter(TestCase):
+    method_config = "./tests/resources/config/method_config.yml"
+    structure_config = "./tests/resources/config/structure_config.yml"
+    project_root = os.getcwd()
+
+    container = {
+            "Maths": {
+                "sum_column": {
+                    "params": {"column_a": "String", "column_b": "String", },
+                    "returns": "String",
+                    "docs": "This function adds the two columns together"
+                },
+                "mulitply": {
+                    "params": {"column_a": "String", "column_b": "String", },
+                    "returns": "String",
+                    "docs": "This function multiplies the two columns together"
+                }
+            },
+            "Estimation": {"ratio": {
+                "params": {"column_a": "String", "column_b": "String", },
+                "returns": "String",
+                "docs": "This function does ratio estimation"
+            }}
+        }
+    structure = {
+            "python": {
+                "path": "/test_dir/python/path/",
+                "template": "python.txt",
+                "file_name_format": "{}.py"
+            },
+            "scala": {
+                "path": "/test_dir/scala/path/",
+                "template": "scala.txt",
+                "file_name_format": "{}.scala"
+            },
+        }
+
+    structure_class = [Structure(project_root, "./test_dir/python/path/", "python.txt", ".py"),
+                           Structure(project_root, "./test_dir/scala/path/", "scala.txt", ".scala")]
+
+    def test_read_configs(self):
```

Comment:
test_read_container_config ?

Or combine this with test_read_structure_config

Reply:
changed the name